src/app/components/PlayersBoard.ts

```diff
@@ -0,0 +1,94 @@
+import { Render } from '../utils/Render';
+import { flatten } from 'lodash';
+import { Herd } from '.././logic/Herd';
+
+export class PlayersBoard {
+  /* returns player's board with player's name, avatar and herd */
+  renderPlayersBoard(
+    playersChosenAvatarPath: string,
+    playersChosenName: string,
+    playersColor: string,
```

Comment:
Colors needs to be added somewhere as default, or perhaps random or get chosen from a few given types. To be analyzed separetely.